views/index.erb

```diff
@@ -1,8 +1,31 @@
 <h1><%= @title %></h1>
 
 Output generated: <%= Time.now %>
+
 <pre>
 $ <%= @command %>
-
-<%= @output %>
 </pre>
+
+<table class="table table-bordered">
+  <tr>
+    <th>USER</th>
+    <th>PID</th>
+    <th>%CPU</th>
+    <th>%MEM</th>
+    <th>VSZ</th>
+    <th>RSS</th>
+    <th>TTY</th>
+    <th>STAT</th>
+    <th>START</th>
+    <th>TIME</th>
+    <th>COMMAND</th>
+  </tr>
+  <% @output.each do |row| %>
+  <tr>
+    <% row.each do |value| %>
+      <td><%= value %></td>
+    <% end %>
+  </tr>
```

Comment:
We should talk about this. I think trying to coerce this into a table in the first place starts to get us away from our original intent of building a simple app that we can build a tutorial around. Adding objects as structs and optimizing the regex makes this more complicated and will require lengthier discussion in a tutorial, which I think would add to the confusion in app development. 

If the goal is to provide a production-level application that can be added to the OOD deployment, great, lets polish this up. But as a simple app for a tutorial, I think the current state is  already more complicated than it needs to be.